modules/workflows/Nextflow_workflows/Modules/metadata_anonymization.py

```diff
@@ -0,0 +1,91 @@
+from HITI_anon_internal.Anon import EmoryAnon
+import sys
+import pandas as pd
+
+def anonymization(metadata, Anon):
+    metadata['AccessionNumber'] = metadata.AccessionNumber.map(str)
+    metadata['PatientID'] = metadata.PatientID.map(int)
+    metadata['StudyDate_modified'] = pd.to_datetime(metadata['StudyDate'], format='%Y%m%d')
+
+    mask = metadata.AccessionNumber.str.len()==16
+    metadata = metadata.loc[mask]
+    metadata.reset_index(inplace=True, drop=True)
+
+    metadata['StudyDate'] = metadata.StudyDate.astype(str)
+    metadata['StudyDate_formatted'] = (metadata.StudyDate.str.slice(0,4))+'-'+(metadata.StudyDate.str.slice(4,6))+'-'+(metadata.StudyDate.str.slice(6,8))
```

Comment:
Are you planning to use StudyDate_formatted somewhere else? If not, remove the redundancy.